packages/transport/test/testHelpers/TestUtil.ts

```diff
@@ -36,8 +38,10 @@ import {
     Transport,
     TransportLoggerFactory
 } from "../../src";
```

Comment:
No Import from Src

Reply:
ehm, actually we import from src in tests ;)